handlers/homepage/handler.go

```diff
@@ -2,9 +2,9 @@
 package homepage
 
 import (
+	homepage "github.com/ONSdigital/dp-frontend-models/model/homepage"
```

Comment:
Do you really need to alias `homepage`? Am I missing something?

Reply:
Yer I had rathered not but we have an unfortunate conflict, homepage was using the geography homepage and every time you remove the alias it will switch back to waning the geography one. Even deleted the file to try to get around it and recreate it after a build there might be a better way to force it to use the right model but this seemed to do the trick